### PR TITLE
Bugfix/comp services see all inputs

### LINKED ISSUE #1473: logs are not uploaded from sidecar with services 1.0.0
logs are correctly pushed to rabbit but no file is generated and the log.zip is thus empty.

### LINKED ISSUE #1723: Running some services is too slow
### Notes
This issue might be related with how we currently handle some things inside the services themselves. I will describe the scenario first. 

Manifesting this issue is the `sleeper` service. All computational services should have the same issue as well. The dynamic services might inherit the same problem.

### What actually happens

I was able to observe this happening only on master.

**How to reproduce:**

- create a new study
- add the service `sleeper` not the gpu or mpi version of it
- press run button and look at the service logs

It will try to change permissions to around 5000 files in the input folder and 300 files in the output folder. These operations take a lot of time.

If you start a new terminal in the running service you can see there are a lot of directories inside the input folder.

Screenshot 2020-08-20 at 10 41 16

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?

- sidecar now bind mount only a subfolder of the respective named volumes (input,output,log) by inspecting the said volumes to get their mountpoints on the hosts (**NOTE**: this might break running comp services locally on a Windows machine with WSL1)
- Therefore the computational services that change the user/groups of each files should go faster since they won't have access to anything else than their defined subfolder

Bonus:
- fixes #1473 
- fixes some invalid syntax when creating aio_pika callbacks
## Related issue number

fixes #1723

## Checklist

- [ ] Did you change any service's API? Then make sure to bundle document and upgrade version (``make openapi-specs``, ``git commit ...`` and then ``make version-*``)
- [ ] Unit tests for the changes exist
- [ ] Runs in the swarm
- [ ] Documentation reflects the changes
- [ ] New module? Add your github username to [.github/CODEOWNERS](.github/CODEOWNERS)

### CODE REVIEW

**Review by odeimaiz — APPROVED** [review]

File: services/sidecar/src/simcore_service_sidecar/utils.py [hunk 1]
```diff
@@ -134,3 +136,20 @@ def assemble_celery_app(task_default_queue: str, rabbit_config: RabbitConfig) ->
     )
     app.conf.task_default_queue = task_default_queue
     return app
+
+
+async def get_volume_mount_point(volume_name: str) -> str:
+    try:
+        docker_client: aiodocker.Docker = aiodocker.Docker()
+        volume_attributes = await DockerVolume(docker_client, volume_name).show()
+        VOLUME_MOUNTPOINT = "Mountpoint"
```
[the remark this answers is not in the document]
Comment (pcrespov) [reply services/sidecar/src/simcore_service_sidecar/utils.py:145]: SUGGESTION: i find this more pythonic
```python

     try:

        docker_client: aiodocker.Docker = aiodocker.Docker()
        volume_attributes = await DockerVolume(docker_client, volume_name).show()
        return volume_attributes["Mountpoint"]

     except aiodocker.exceptions.DockerError:
            logger.exception("...")
            raise SidecarException("other message") from err

     except KeyError as err:
            raise SidecarException("other message") from err
      
```

File: services/sidecar/src/simcore_service_sidecar/executor.py [hunk 2]
```diff
@@ -3,39 +3,40 @@
...
+    output_folder: Optional[Path] = None
+    log_folder: Optional[Path] = None
 
     @classmethod
     def from_task(cls, task: aiopg.sa.result.RowProxy):
         return cls(
-            Path.home() / f"input/{task.job_id}",
-            Path.home() / f"output/{task.job_id}",
-            Path.home() / f"log/{task.job_id}",
+            config.SIDECAR_INPUT_FOLDER / f"{task.job_id}",
```
Comment (pcrespov) [inline services/sidecar/src/simcore_service_sidecar/executor.py:37]: do you really need the f-string? ``task.job_id`` is not a string on its own?

File: services/sidecar/src/simcore_service_sidecar/executor.py [hunk 3]
```diff
@@ -232,9 +241,11 @@ async def _run_container(self):
                 "Init": True,
                 "AutoRemove": False,
                 "Binds": [
-                    f"{config.SIDECAR_DOCKER_VOLUME_INPUT}:/input",
-                    f"{config.SIDECAR_DOCKER_VOLUME_OUTPUT}:/output",
-                    f"{config.SIDECAR_DOCKER_VOLUME_LOG}:/log",
+                    # NOTE: the docker engine is mounted, so only named volumes are usable. Therefore for a selective
+                    # subfolder mount we need to get the path as seen from the host computer (see https://github.com/ITISFoundation/osparc-simcore/issues/1723)
```
Comment (pcrespov) [inline services/sidecar/src/simcore_service_sidecar/executor.py:245]: excellent!

**Review by pcrespov — CHANGES_REQUESTED** [review]
some minor

File: services/sidecar/src/simcore_service_sidecar/executor.py (hunk 2 above)
Comment (sanderegg) [reply services/sidecar/src/simcore_service_sidecar/executor.py:37]: just why is this an issue? even if it was a string I mean

File: services/sidecar/src/simcore_service_sidecar/utils.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (sanderegg) [reply services/sidecar/src/simcore_service_sidecar/utils.py:145]: ok

File: services/sidecar/src/simcore_service_sidecar/executor.py (hunk 2 above)
Comment (pcrespov) [reply services/sidecar/src/simcore_service_sidecar/executor.py:37]: not an issue , just a question ;-)

**Review by pcrespov — APPROVED** [review]

**Comment by pcrespov** [comment]
It would be good to increase testing on this ...

**Pull request merged**